### PR TITLE
Adding before-create-topic-button plugin-outlet

### PR DESCRIPTION
**Pull request by marielaSAG**
on tag pages

### CODE REVIEW

File: app/assets/javascripts/discourse/app/templates/tags/show.hbs [hunk 1]
```diff
@@ -34,6 +34,13 @@
             }}
           {{/unless}}
         {{/if}}
+        
+        {{plugin-outlet name="before-create-topic-button"
+        args=(hash
```
Comment (eviltrout) [inline app/assets/javascripts/discourse/app/templates/tags/show.hbs:39]: This is super minor but the indentation here is off. Could you line up `args` with `plugin-outlet`?

Comment (marielaSAG) [reply app/assets/javascripts/discourse/app/templates/tags/show.hbs]: Sure, thanks, I did that.

Comment (eviltrout) [reply app/assets/javascripts/discourse/app/templates/tags/show.hbs]: Great, we've tagged it for merging after the next release.

**Comment by SamSaffron** [comment]
Thanks, merged it in!

**Pull request merged**